scripts/e2etests/VSUtils.ps1

```diff
@@ -203,10 +203,15 @@ function DowngradeVSIX
     Write-Host "$VSIXInstallerPath" -Wait -PassThru -NoNewWindow -ArgumentList "/q /a /d:$vsixID"
     $p = start-process "$VSIXInstallerPath" -Wait -PassThru -NoNewWindow -ArgumentList "/q /a /d:$vsixID"
 
-    if ($p.ExitCode -ne 0)
+    if($p.ExitCode -eq 2001)
```

Comment:
What's the happy case here? 
@mishra14 

What happens when it actually downgrades?